cmd/shim/main.go

```diff
@@ -0,0 +1,235 @@
+package main
```

Comment:
small nit: was this file `mv`'d instead of `git mv`? Or was just `git confused

Doesn't matter, but will make your rebase on the other PR harder

Reply:
Oh yeah I just `mv`'d probably. I'll see if it's possible to retroactively `git mv` it somehow.